packages/features/calendar-cache/calendar-cache.repository.ts

```diff
@@ -11,30 +11,21 @@ import { getTimeMax, getTimeMin } from "./lib/datesForCache";
 
 const log = logger.getSubLogger({ prefix: ["CalendarCacheRepository"] });
 
-/** Enable or disable the expanded cache. Enabled by default. */
-const ENABLE_EXPANDED_CACHE = process.env.ENABLE_EXPANDED_CACHE !== "0";
```

Comment:
Removed to avoid possible clash in here

https://github.com/calcom/cal.com/blob/fix/calendar-cache-date-mismatch/packages/app-store/googlecalendar/lib/CalendarService.ts?rgh-link-date=2025-03-29T17%3A13%3A44Z#L1061